poll/public/html/poll_edit.html

```diff
@@ -0,0 +1,96 @@
+{{js_template|safe}}
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+    <form id="poll-form">
+    <ul class="list-input settings-list" id="poll-line-items">
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label poll-setting-label" for="poll-display-name">Display Name</label>
```

Comment:
for future reference, if you make the associated ```<input>``` a child of the ```<label>``` you don't have to worry about the ```for/id``` references.